source/server/management/performance-tweaks.rst

```diff
@@ -0,0 +1,91 @@
+==================
+Performance Tweaks
+==================
+
+Sponge provides several performance enhancing and tweaking options to help you with running a lag-free server, even
+under heavy load.
+
+.. note::
+  While we try to improve the performance of every server, the performance gained depends on your setup. Please run some
+  tests on your server to ensure you configured your Sponge server to get the most out of it!
```

Comment:
No need to mention "server" twice in the sentence, I'd drop out "on your server".